extensions/amp-ad-network-adzerk-impl/0.1/data/0

```diff
@@ -1,5 +1,5 @@
 {
-  "templateUrl": "https://cdn.ampproject.org/ad-template/0001.html",
+  "templateUrl": "https://ampjs.org/ad-template/0001.html",
```

Comment:
Is this hosted by `ampjs.org`?

Reply:
Nope, but neither is this hosted on `cdn.ampproject.org` - it's a fake URL for a fake template